File: C/beginner/1178.c
// https://judge.beecrowd.com/pt/problems/view/1178
// Preenchimento de Vetor III

// Libraries
#include <stdio.h>

// Main
int main(){
    // Variables
    double N, vetor[100];

    // Input
    scanf("%lf", &N);

    // Fill the array with the values
    vetor[0] = N;
    for (int i = 1; i < 100; i++){
        vetor[i] = vetor[i - 1] / 2;
    }

    // Output
    for (int i = 0; i < 100; i++){
        printf("N[%d] = %.4lf\n", i, vetor[i]);
    }

    return 0;
}

// By: Fernando Serra
// https://github.com/fernandosserra

File: Python/beginner/1178.py
# https://judge.beecrowd.com.br/problems/view/1178
# Preenchimento de Vetor III

# Read the first input value
usrInput = float(input())

# Initialize the vector with the first input value and fill it with halved values
vector = [usrInput]
for i in range(1, 100):
    vector.append(vector[i - 1] / 2)

# Print the vector values formatted to four decimal places
for i in range(100):
    print(f"N[{i}] = {vector[i]:.4f}")
    
# By: Fernando Serra
# https://github.com/fernandosserra